namelists/fesom2/2.1/namelist.config

```
@@ -38,7 +38,7 @@ cartesian=.false.
 fplane=.false.
 cyclic_length=360       ![degree]
 rotated_grid=.true.     !option only valid for coupled model case now
-force_rotation=.true.
+force_rotation=.false.
```

```suggestion
force_rotation=.true.
```
@koldunovn can you confirm that I am understanding correctly? Default FESOM mesh is unrotated, and `force_rotation` should be set to `.true.` by default?

---

Yes, by default meshes in FESOM2.1 are unrotated (or rather not rotated :)) and `force_rotation` therefore should be `true`.